hack/build-go.sh

```diff
@@ -10,7 +10,11 @@ STARTTIME=$(date +%s)
 OS_ROOT=$(dirname "${BASH_SOURCE}")/..
 source "${OS_ROOT}/hack/common.sh"
 source "${OS_ROOT}/hack/util.sh"
-os::log::install_errexit
+source "${OS_ROOT}/hack/lib/util/trap.sh"
```

Comment:
1. not everyone is going to need everything
2. compartmentalizing our support functions helps people understand them
3. our "lets put everything in `common.sh`" approach failed as soon as the file got so long that multiple versions of the same function started showing up with different implementations.